server/views/pages/create/additionalConditions.njk

```diff
@@ -0,0 +1,123 @@
+                                })
+                            }
+                        }
+                    ]
+                }) }}
+
+                {% set restrictionOfResidencyHtml %}
+                    {{ govukRadios({
+                        name: "supplementary-data[restriction-of-residency-condition-id-0][gender]",
+                        fieldset: {
+                            legend: {
+                                text: "Select the relevent text"
+                            }
+                        },
+                        items: [
+                            {
+                                value: "any",
+                                text: "any"
+                            },
+                            {
+                                value: "any female",
+                                text: "any female"
+                            },
+                            {
+                                value: "any male",
+                                text: "any male"
+                            }
+                        ]
+                    }) }}
+
+                    {{ govukRadios({
+                        classes: "govuk-radios--inline",
+                        name: "supplementary-data[restriction-of-residency-condition-id-0][age]",
+                        fieldset: {
+                            legend: {
+                                text: "Select the relevent age"
+                            }
+                        },
+                        items: [
+                            {
+                                value: "16",
+                                text: "16 years"
+                            },
+                            {
+                                value: "18",
+                                text: "18 years"
+                            }
+                        ]
+                    }) }}
+                {% endset %}
+
+                {{ govukCheckboxes({
+                    name: "additional-conditions",
+                    fieldset: {
+                        legend: {
+                            text: "Restriction of residency",
+                            isPageHeading: true,
+                            classes: "govuk-fieldset__legend--l"
+                        },
+                        classes: "govuk-!-margin-top-6"
+                    },
+                    items: [
+                        {
+                            value: "restriction-of-residency-condition-id-0",
+                            text: "Not to reside (not even to stay for one night) in the same household as [ANY / ANY FEMALE / ANY MALE] child under the age of [INSERT AGE] without the prior approval of your supervising officer.",
+                            conditional: {
+                                html: restrictionOfResidencyHtml
+                            }
+                        }
+                    ]
+                }) }}
```

Comment:
Are we still missing a lot of the questions? I think there were a lot more in the prototype, but a good start to get some of them in.

Reply:
Included a subset of the conditions as hardcoded html with an example of what the payload could look like. We would eventually be migrating this to html-as-config I think? So didn't hard code the whole thing for now but happy to do so in the short term